### PR TITLE
Add azure_cache as storage policy to tests

### PR DESCRIPTION
**Pull request by SmitaRKulkarni**
### Changelog category (leave one):
- Not for changelog (changelog entry is not required)

cc @kssenii

### CODE REVIEW

**Comment by robot-ch-test-poll2** [comment]
*This is an automated comment for commit 7f34af467e955a363debb3575b45f30cd60b64f3 with description of existing statuses. It's updated for the latest CI running*

[⏳ Click here](https://s3.amazonaws.com/clickhouse-test-reports/59943/7f34af467e955a363debb3575b45f30cd60b64f3/ci_running.html) to open a full report in a separate page

<table>
<thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody>
<tr><td>CI running</td><td>A meta-check that indicates the running CI. Normally, it's in <b>success</b> or <b>pending</b> state. The failed status indicates some problems with the PR</td><td><a href="https://s3.amazonaws.com/clickhouse-test-reports/59943/7f34af467e955a363debb3575b45f30cd60b64f3/ci_running.html">⏳ pending</a></td></tr>
<tbody>
</table>

File: tests/config/install.sh [hunk 1]
```diff
@@ -168,7 +168,7 @@ ARM="aarch64"
 OS="$(uname -m)"
 if [[ -n "$EXPORT_S3_STORAGE_POLICIES" ]]; then
     echo "$OS"
-    if [[ "$USE_DATABASE_REPLICATED" -eq 1 ]] || [[ "$OS" == "$ARM" ]]; then
```
Comment (SmitaRKulkarni) [inline tests/config/install.sh:171]: @kssenii : do you happen to know why azure is disabled for replicated database ?

Comment (kssenii) [reply tests/config/install.sh:171]: Probably a problem with ports https://github.com/ClickHouse/ClickHouse/pull/36171/commits/52832ea7882311f8caea58c39b94aaf9add08b56#diff-3575b7ff67cdd826106f096c6d432a807060a711e173ed06ced7e608c4b0c058R21-R25 when running run.sh.

**Comment by kssenii** [comment]
@SmitaRKulkarni please check failing tests: `Unknown storage policy 'azure_cache'`

**Review by kssenii — APPROVED** [review]

**Pull request merged**